fuzzy/docker/apps/fsl/Dockerfile.libmath.fsl

```diff
@@ -0,0 +1,7 @@
+FROM gkiar/fuzzy:ubuntu14.04-libmath
+
+RUN apt-get update && apt install -y wget
+
+ADD apps/fsl/neurodebian-travis.sh /tmp
```

Comment:
This would just have to be updated to point to the new location of the install script, otherwise, perfect!